package.json

```diff
@@ -1,96 +1,51 @@
 {
-  "name": "angular2-jsonapi",
-  "version": "7.1.2",
-  "description": "A lightweight Angular 2 adapter for JSON API",
+  "name": "angular2-jsonapi-new",
+  "version": "0.0.0",
+  "private": true,
```

Comment:
Since this is the Angular "workspace" `package.json`, it should be private. The library `package.json` is set to public, and that's the one that is published.
I'll rename the workspace so it is clearer.